04 - Guides, Workflows, & Courses/Community Talks/Obsidian and TTRPG.md

```diff
@@ -24,6 +24,6 @@ The order of speakers is as follows:
 
 %% Hub footer: Please don't edit anything below this line %%
 
-# This note in GitHub
+# This Note in GitHub
```

Comment:
For info only, as this is harmless as it will get overwritten - just to point out for future reference, the comment above requests to not edit the comment

Reply:
Ah, I missed that.